crucible-llvm/src/Lang/Crucible/LLVM/Translation/Constant.hs

```diff
@@ -1090,3 +1091,6 @@ transConstantExpr expr = case expr of
  where
  badExp :: String -> m a
  badExp msg = throwError $ unlines [msg, show expr]
+
+testBreakpointFunction :: String -> Maybe String
```

Comment:
Seems a little weird to put this here?

Reply:
I'm not sure... maybe in the main Translation module? It just seems strange to have it defined, but not used at all here.